frontend/src/components/common/TabNavigation.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Tab from "./Tab";
+
+class TabNavigation extends Component {
+  static propTypes = {
+    tabSpecs: PropTypes.array.isRequired,
+    curTab: PropTypes.number.isRequired
```

Comment:
can we spell out `current` - I find this hard to read.

Reply:
Fixed in earlier merge